index.rst

```diff
@@ -15,5 +10,265 @@ Change Record
+---------------
+
+Two general rules are applied for the DM Team labor when determining whether an activity should be
+treated as LOE or non-LOE:
+
+* All activities that have a clear deliverable are tracked through non-LOE. Examples of non-LOE
+  activities include writing a new piece of software, purchasing new equipment, or adding new
+  machines to the cluster.
+
+* Activities that can not be predicted, as well as short, recurring routine activities are considered LOE.
+  Examples of LOE activities include replacing faulty disk drive, cluster maintenance, or writing monthly
+  status report.
+
+For planning purposes, 70% of available time of each software developer is assumed to be available
+for "pure, uninterrupted coding", and the remaining 30% is considered as "overhead", and tracked
+explicitly as LOE.
+
+A small fraction of all DM labor is performed by Contractors. The contractor labor is managed as LOE,
+directly in dollars.
+
+People
+======
+
+* DM Project Manager responsible for overall plan, schedule and priorities.
+
+* Work divided into smaller pieces, through WBS.
+
+* Each piece has a dedicated technical manager.
+
+* Main responsibilities of technical managers:
+
+  * Assemble the team capable of delivering work scoped through the WBS on-time and within budget.
+    Provide daily technical management and leadership for the team, monitor and optimize team performance.
+
+  * Work closely with the DM Project Manager on defining short and long-term plan and schedule for
+    their teams. Direct the execution of their team's plan, ensuring the team delivers on-time and within budget.
+
+  * Report group's activities as required, including reporting to the Earned Value Management System (EVMS)
+    used by LSST, and providing input for monthly status reports.
+
+* In a sense, DM PM "owns" the overall baseline plan (PPs, milestones), and technical manager "own" epics.
+
+
+Tools
+=====
+
+Master copy of the Baseline Plan, which includes all planning packages and all milestones is captured
+in Primavera. This is always the authoritative source of truth for the Baseline Plan.
+
+Information about milestones is also kept in JIRA DM Baseline Plan (DMBP) project, and is periodically
+synchronized with Primavera. Having milestones in JIRA is in particular useful for expressing blocking
+relations between milestones and epics between different teams.
+
+Master copy of Epics and Stories is kept in JIRA "DM" project. The snapshot for
+epics corresponding to current and upcoming Cycle is taken and kept in Primavera.
+
+Information in JIRA for current Cycle is particularly important, it needs to be
+kept up-to-date and it should reflect the current state of the progress. Progress
+updates for activities that block other teams, in particular when the delivery date
+approaches are expected to be done promptly (depending on urgency, weekly or even daily)
+
+
+The JIRA DMBP serves as an easy-to-use interface TCAMs can use to interact with
```

Comment:
Above, we talked about "technical managers"; here, it's "TCAMs". Are we using these terms interchangeably, or are we maintaining a subtle distinction (some folks provide technical leadership to their teams while not fulfilling the EVMS CAM role)? Maybe worth a word or two to clarify.